odoo/addons/base/tests/test_res_partner_merge.py

```diff
@@ -0,0 +1,105 @@
+from odoo.tests.common import TransactionCase
+from odoo import Command
+
+
+class TestMergePartner(TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.Partner = self.env['res.partner']
+        self.Bank = self.env['res.partner.bank']
+
+        # Create partners
+        self.partner1 = self.Partner.create({'name': 'Partner 1', 'email': 'partner1@example.com'})
+        self.partner2 = self.Partner.create({'name': 'Partner 2', 'email': 'partner2@example.com'})
+        self.partner3 = self.Partner.create({'name': 'Partner 3', 'email': 'partner3@example.com'})
+
+        # Create bank accounts
+        self.bank1 = self.Bank.create({'acc_number': '12345', 'partner_id': self.partner1.id})
+        self.bank2 = self.Bank.create({'acc_number': '54321', 'partner_id': self.partner2.id})
+        self.bank3 = self.Bank.create({'acc_number': '12345', 'partner_id': self.partner3.id})  # Duplicate account number
+
+        # Create references
+        self.attachment1 = self.env['ir.attachment'].create({
+            'name': 'Attachment 1',
+            'res_model': 'res.partner',
+            'res_id': self.partner1.id,
+        })
+        self.attachment2 = self.env['ir.attachment'].create({
+            'name': 'Attachment 2',
+            'res_model': 'res.partner',
+            'res_id': self.partner2.id,
+        })
+        self.attachment_bank1 = self.env['ir.attachment'].create({
+            'name': 'Attachment Bank 1',
+            'res_model': 'res.partner.bank',
+            'res_id': self.bank1.id,
+        })
+        self.attachment_bank2 = self.env['ir.attachment'].create({
+            'name': 'Attachment Bank 2',
+            'res_model': 'res.partner.bank',
+            'res_id': self.bank2.id,
+        })
+        self.attachment_bank3 = self.env['ir.attachment'].create({
+            'name': 'Attachment Bank 2',
+            'res_model': 'res.partner.bank',
+            'res_id': self.bank3.id,
+        })
+
+    def test_merge_partners_without_bank_accounts(self):
+        """ Test merging partners without any bank accounts """
+        partner4 = self.Partner.create({'name': 'Partner 4', 'email': 'partner4@example.com'})
+        partner5 = self.Partner.create({'name': 'Partner 5', 'email': 'partner5@example.com'})
+        wizard = self.env['base.partner.merge.automatic.wizard'].create({
+            'partner_ids': [Command.set([partner4.id, partner5.id])],
+            'dst_partner_id': partner4.id,
+        })
```

Comment:
```suggestion
        wizard = self.env['base.partner.merge.automatic.wizard'].create({})
```
Don't forget to remove the `Command` import :wink:

Reply:
Done!